### PR TITLE
Readme typo fix

### PR DESCRIPTION
**Pull request by SpencerCDixon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -347,8 +347,8 @@ class NullFixture extends React.Component {
 
 const nullWrapper = mount(<NullFixture />) // mount/render/shallow when applicable
 
-expect(wrapper).to.be.blank()
-expect(wrapper).to.be.empty // an alias
+expect(nullWrapper).to.be.blank()
+expect(nullWrapper).to.be.empty // an alias
```
Comment (ayrton) [inline README.md]: Can you amend your commit to do this for the `nullWrapper` under the following section too please.

Comment (SpencerCDixon) [inline README.md]: Oh yeah totally, didn't even see that!

**Comment by SpencerCDixon** [comment]
should be good now @ayrton

**Comment by ayrton** [comment]
Thanks @SpencerCDixon

**Comment by SpencerCDixon** [comment]
no, thank you!  Love the library :+1:

**Pull request merged**